src/js/apps/globals/nav_app.js

```diff
@@ -206,6 +210,7 @@ export default App.extend({
     const patientSearchModal = new PatientSearchModal({
       collection: Radio.request('entities', 'searchPatients:collection'),
       prefillText,
+      isFromAddPatient: !!this.getState('addPatientModal'),
```

Comment:
In talking with Mark, a nice-to-have is the ability to go back to the add patient modal from patient search modal.